packages/next/build/analysis/get-page-static-info.ts

```diff
@@ -32,13 +35,28 @@ export async function getPageStaticInfo(params: {
   isDev?: boolean
   page?: string
 }): Promise<PageStaticInfo> {
-  const { isDev, pageFilePath, nextConfig } = params
+  const { isDev, pageFilePath, nextConfig, page } = params
 
   const fileContent = (await tryToReadFile(pageFilePath, !isDev)) || ''
   if (/runtime|getStaticProps|getServerSideProps|matcher/.test(fileContent)) {
     const swcAST = await parseModule(pageFilePath, fileContent)
     const { ssg, ssr } = checkExports(swcAST)
-    const config = tryToExtractExportedConstValue(swcAST, 'config') || {}
+
+    // default value for config
+    let config: any = {}
+    try {
+      config = extractExportedConstValue(swcAST, 'config')
+    } catch (e) {
+      if (e instanceof UnsupportedValueError) {
+        // `export config` is found, but can't extract the value of it
+        Log.warn(
+          `You have exported a \`config\` field in "${
+            page || pageFilePath
+          }" that Next.js can't recognize, so it will be ignored`
+        )
+      }
+      // `export config` doesn't exist or other unknown error, silence them
+    }
```

Comment:
@balazsorban44 I have added a warning here about unrecognized `export config`.

Other kinds of errors will be silently ignored, just as before.